app/uk.gov.hmrc.helptosavestridefrontend/connectors/HelpToSaveConnector.scala

```diff
@@ -182,6 +188,32 @@ class HelpToSaveConnectorImpl @Inject() (http:                              WSHt
     })
   }
 
+  override def getEnrolmentStatus(nino: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Result[EnrolmentStatus] = {
+
+    EitherT[Future, String, EnrolmentStatus](http.get(enrolmentStatusUrl).map[Either[String, EnrolmentStatus]] { response ⇒
+      response.status match {
+        case OK ⇒
+          val result = response.parseJson[EnrolmentStatus]
+          result.fold({
+            e ⇒
```

Comment:
do we need to worry about PagerDuty alerts here at all ?

Reply:
Adding...